common/types/block_test.go

```diff
@@ -88,8 +89,11 @@ func TestBytesToNodeID(t *testing.T) {
 	r.Error(err, "Expected error converting too-short byte array to NodeID")
 
 	// Test too long
-	var y [65]byte
-	rand.Read(y[:])
-	_, err = BytesToNodeID(y[:])
-	r.Error(err, "Expected error converting too-long byte array to NodeID")
+	longSlice := append(bytes, x[:]...)
+	r.Greater(len(longSlice), len(bytes))
+	reversed, err = BytesToNodeID(longSlice)
+	r.NoError(err, "Error converting too-long byte array to NodeID")
+	r.Equal(nodeID1.Key, reversed.Key, "NodeID Key does not match for too-long")
+	r.Equal([]byte{}, reversed.VRFPublicKey, "VRF Key is not empty for too-long")
```

Comment:
is there a difference between `[]byte{}` and `[]byte(nil)` (which you use below)?